modules/rosa-sts-interactive-ocm-and-user-role-creation-mode-options.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started_sts/rosa_creating_a_cluster_with_sts/rosa-sts-interactive-mode-reference.adoc
+
+:_content-type: REFERENCE
+[id="rosa-sts-interactive-ocm-and-user-role-creation-mode-options_{context}"]
+= Interactive OCM and user role creation mode options
+
+Before you can use {cluster-manager-first} to create {product-title} (ROSA) clusters that use the AWS Security Token Service (STS), you must associate your AWS account with your Red Hat organization by creating and linking the OCM and user roles. You can enable interactive mode by specifying the `--interactive` option when you run `rosa create ocm-role` or `rosa create user-role`.
```

Comment:
If I'm not mistaken this is only needed if you are going to use the console (UI)
Someone else should probably confirm this

Reply:
Good catch! I'll make that explicitly clear in this line.

Reply:
On review, I think this is already clear given that the first sentence explicitly refers to using OpenShift Cluster Manager to create ROSA clusters. Given this, I will leave this as is.